math/ruy/CMakeLists.txt

```diff
@@ -44,17 +44,21 @@ if(CONFIG_MATH_RUY)
     endif()
   endif()
 
+  # cmake-format: off
```

Comment:
why add

Reply:
`cmake-format` tool cannot recognize the comment format. You need to turn off format detection here.

Reply:
but why not update the comment to make cmake-format happy

Reply:
The following line is a comment line, perhaps I should remove the logic below without any reservation?

Reply:
yes, it's better to remove the code which doesn't work as expect.

Reply:
Done